### PR TITLE
chore: Add TrackTile settings

### PR DESCRIPTION
**Pull request by namick**
## Changes

  - Add TrackTile settings

## Screenshots

Screenshot 2023-04-14 at 5 35 52 PM

### CODE REVIEW

File: src/navigators/HomeStack.tsx [hunk 1]
```diff
@@ -6,12 +6,18 @@ import { AppTileScreen } from '../screens/AppTileScreen';
...
+import { TrackerValuesContext } from '../components/TrackTile/main';
+import TrackTileSettingsScreen from '../screens/TrackTileSettingsScreen';
+import { t } from '@i18n';
 
 export type HomeStackParamList = {
   Home: undefined;
   'tiles/AppTile': { appTile: AppTile };
   'tiles/CustomAppTile': { appTile: AppTile };
   'tiles/TrackTile': { tracker: any; valuesContext: any };
+  'tiles/trackTileSettings': {
```
Comment (sternetj) [inline src/navigators/HomeStack.tsx:18]: The casing is different than the others. Do we want to make that consistent or are we switching to snack-case?

File: src/screens/TrackTileSettingsScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+import { NativeStackScreenProps } from '@react-navigation/native-stack';
+import { HomeStackParamList } from '../navigators/HomeStack';
+import { ManageTrackers } from '../components/TrackTile/ManageTrackers/ManageTrackers';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+  },
+});
```
Comment (sternetj) [inline src/screens/TrackTileSettingsScreen.tsx:12]: Should this use are style system?

**Review by sternetj — APPROVED** [review]

File: src/screens/TrackTileSettingsScreen.tsx (hunk 2 above)
Comment (namick) [reply src/screens/TrackTileSettingsScreen.tsx:12]: Yep, coming up in the next PR

File: src/navigators/HomeStack.tsx (hunk 1 above)
Comment (namick) [reply src/navigators/HomeStack.tsx:18]: Good catch.  Looking at this now, are we attempting to imbue semantic meaning into these route names?  Maybe we should have them align with their stack location.

**Pull request merged**